service/javadsl/client/src/main/java/com/lightbend/lagom/javadsl/client/CircuitBreakersConverter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.client;
+
+import scala.compat.java8.FutureConverters;
+import scala.concurrent.Future;
+import scala.runtime.AbstractFunction0;
+
+import java.util.concurrent.CompletionStage;
+import java.util.function.Supplier;
+
+public class CircuitBreakersConverter {
+
+  /**
+   * Temporary converter utility to convert {@link com.lightbend.lagom.internal.client.CircuitBreakers} to {@link CircuitBreakersPanel}
+   */
+  public static CircuitBreakersPanel toJavadslCircuitBreakerInvoker(com.lightbend.lagom.internal.client.CircuitBreakers circuitBreakers) {
+    return new CircuitBreakersPanel() {
```

Comment:
What if `CircuitBreakersPanelImpl` had a package-private constructor that takes a `CircuitBreakersPanelInternal`, and `CircuitBreakers.circuitBreakersPanel` were `private[lagom]`?

Then, assuming that this class moves to `com.lightbend.lagom.internal.javadsl`, this method could just `return new CircuitBreakersPanelImpl(circuitBreakers.circuitBreakersPanel)`.

Alternatively, `CircuitBreakers` could extend `CircuitBreakersPanelInternal` and then could be passed directly.

I generally favor composition to inheritance, too, but in this case I think it might make sense to extend. `CircuitBreakers` literally is the same functionally as `CircuitBreakersPanelInternal`, and only exists separately for backwards-compatibility reasons. In fact, the only reason to even extract `CircuitBreakersPanelInternal` is so we can hang a `@deprecated` sign on `CircuitBreakers` while still reusing the functionality.